modules/installation-creating-aws-worker.adoc

```diff
@@ -105,76 +105,7 @@ directory. This value is the long string with the format
 <14> Specify the `WorkerInstanceProfile` parameter value from the output of
 the CloudFormation template for the security group and roles.
 <15> The type of AWS instance to use for the control plane machines.
-<16> Allowed values:
-* `m4.large`
-* `m4.xlarge`
-* `m4.2xlarge`
-* `m4.4xlarge`
-* `m4.10xlarge`
-* `m4.16xlarge`
-* `m5.large`
-* `m5.xlarge`
-* `m5.2xlarge`
-* `m5.4xlarge`
-* `m5.8xlarge`
-* `m5.12xlarge`
-* `m5.16xlarge`
-* `m5a.large`
-* `m5a.xlarge`
-* `m5a.2xlarge`
-* `m5a.4xlarge`
-* `m5a.8xlarge`
-* `m5a.12xlarge`
-* `m5a.16xlarge`
-* `c4.large`
-* `c4.xlarge`
-* `c4.2xlarge`
-* `c4.4xlarge`
-* `c4.8xlarge`
-* `c5.large`
-* `c5.xlarge`
-* `c5.2xlarge`
-* `c5.4xlarge`
-* `c5.9xlarge`
-* `c5.12xlarge`
-* `c5.18xlarge`
-* `c5.24xlarge`
-* `c5a.large`
-* `c5a.xlarge`
-* `c5a.2xlarge`
-* `c5a.4xlarge`
-* `c5a.8xlarge`
-* `c5a.12xlarge`
-* `c5a.16xlarge`
-* `c5a.24xlarge`
-* `r4.large`
-* `r4.xlarge`
-* `r4.2xlarge`
-* `r4.4xlarge`
-* `r4.8xlarge`
-* `r4.16xlarge`
-* `r5.large`
-* `r5.xlarge`
-* `r5.2xlarge`
-* `r5.4xlarge`
-* `r5.8xlarge`
-* `r5.12xlarge`
-* `r5.16xlarge`
-* `r5.24xlarge`
-* `r5a.large`
-* `r5a.xlarge`
-* `r5a.2xlarge`
-* `r5a.4xlarge`
-* `r5a.8xlarge`
-* `r5a.12xlarge`
-* `r5a.16xlarge`
-* `r5a.24xlarge`
-* `t3.large`
-* `t3.xlarge`
-* `t3.2xlarge`
-* `t3a.large`
-* `t3a.xlarge`
-* `t3a.2xlarge`
+<16> Suggested values: `<InstanceType>.large`, where `InstanceType` could be: `m5`,`m6i`, etc
```

Comment:
same as https://github.com/openshift/openshift-docs/pull/43558/files#r840312094

Reply:
@yunjiang29 updated to generic value as our conversation on Slack.